vars/karateFlow.groovy

```diff
@@ -72,4 +72,30 @@ def call(params) {
             }
         }
     }
+    
```

Comment:
Please define a separate function with appropriate list of parameters. To make it usable with all another tests.

Reply:
Will be realized in task regarding Cypress tests notifs.